src/test/ui/drop/issue-90752-raw-ptr-shenanigans.rs

```diff
@@ -0,0 +1,41 @@
+// run-pass
```

Comment:
Maybe instead of checking that we are getting the wrong result we should just have `ignore-test FIXME(#90788)`

Reply:
If someone (including me) were to change the behavior of this test, they should be made aware.